OasisDEX/oasis-borrow
# Fixed import warning regarding uniswap token list

# [Fixed import warning regarding uniswap token list](https://shortcutLink)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- fixed import warning regarding uniswap token list
  
## How to test 🧪
  <Please explain how to test your changes>

> - check if uniswap uses correct token list
> - run yarn start and check if following warning messages are still visible

![image](https://user-images.githubusercontent.com/39706811/171345829-2d1adc4d-d272-4408-a0ad-72523787ec44.png)
